.github/workflows/release.yml

```diff
@@ -54,4 +54,8 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        gh release create $ORT_SERVER_VERSION --notes-file RELEASE_NOTES.md
+        PRERELEASE_ARG=""
+        if [[ "$ORT_SERVER_VERSION" == *RC* ]]; then
```

Comment:
Should the pattern be `-RC` for safety? Or even `[[ "$ORT_SERVER_VERSION" =~ "-RC[0-9]+" ]]` to align a bit with line 7?